test/internal/helpers/kubectl/helpers.go

```diff
@@ -244,9 +235,38 @@ func RunSensitiveKubectlCommand(t *testing.T, kubectlOptions *k8s.KubectlOptions
 func ConfigureElasticBackup(t *testing.T, cluster helpers.Cluster, clusterName, inputVersion string) {
 	t.Logf("[ELASTICSEARCH] Configuring Elasticsearch backup for cluster %s", cluster.ClusterName)
 
+	// Replace dots with dashes in the version string.
 	version := strings.ReplaceAll(inputVersion, ".", "-")
 
-	output, err := k8s.RunKubectlAndGetOutputE(t, &cluster.KubectlNamespace, "exec", "camunda-elasticsearch-master-0", "--", "curl", "-XPUT", "http://localhost:9200/_snapshot/camunda_backup", "-H", "Content-Type: application/json", "-d", fmt.Sprintf("{\"type\": \"s3\", \"settings\": {\"bucket\": \"%s-elastic-backup\", \"client\": \"camunda\", \"base_path\": \"%s-backups\"}}", clusterName, version))
+	// Determine if Teleport mode is enabled.
+	teleportEnabled := false
+	if teleportStr, ok := os.LookupEnv("TELEPORT"); ok {
+		if parsed, err := strconv.ParseBool(teleportStr); err == nil {
+			teleportEnabled = parsed
+		} else {
+			t.Fatalf("[ELASTICSEARCH] Failed to parse TELEPORT env var: %v", err)
+		}
+	}
```

Comment:
The following is just a suggestion and preference, just a quick idea but feel free to dismiss it.

we could align it with the rest of the code and use the helper function to retrieve the value?

https://github.com/camunda/c8-multi-region/blob/main/test/internal/helpers/helpers.go#L27-L33

```
teleportEnabled := helpers.GetEnv("TELEPORT", "false")
```

simplified everything a bit since you don't have to convert to boolean and since you control the default value, anything defining `TELEPORT` would be true.

```
if teleportEnabled != "false" {
....
}
```